packages/search-ui-workplace-search-connector/src/WorkplaceSearchAPIConnector.ts

```diff
@@ -157,6 +166,14 @@ class WorkplaceSearchAPIConnector {
       isLoggedIn: !!this.accessToken || false
     };
 
+    this.actions = {
+      logout: () => {
+        this.accessToken = null;
+        this.state.isLoggedIn = false;
```

Comment:
i've forgotten now - how does the connector state sync with SearchDriver state and trigger an view update?